core/core_integrations/messenger.js

```diff
@@ -0,0 +1,198 @@
+var fs = require('fs'),
+	path = require('path'),
+	express = require('express'),
+	bodyParser = require('body-parser'),
+	request = require('request'),
+	https = require('https'),
+	shim = require('../shim.js'),
+	http = null,
+	api = null;
+
+var getNameForId = function(id, callback) {
+	if (!exports.config.users) {
+		exports.config.users = {};
+	}
+	
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://www.codacy.com/app/theprojectshop/Kassy/file/2547584594/issues/source?bid=3282987&fileBranchId=3288129#l15)